main.go

```diff
@@ -47,18 +48,16 @@ func displayUnknown(name string) {
 	}
 }
 
-func display(name string, ref *backvendor.Reference) {
-	fmt.Print(name)
-	if ref.Rev != "" {
-		fmt.Print("@", ref.Rev)
-	}
-	if ref.Tag != "" {
-		fmt.Print(" =", ref.Tag)
-	}
-	if ref.Ver != "" {
-		fmt.Print(" ~", ref.Ver)
+func display(template string, name string, ref *backvendor.Reference) {
+	var builder strings.Builder
+	builder.WriteString(name)
+	var tmpl, err = backvendor.Display(template)
+	if err != nil {
+		log.Warningf("Error parsing supplied template, using default. %s ", err)
```

Comment:
I'd rather just abort at this point (`log.Fatalf`) rather than falling back to the default template.